rocAL/CMakeLists.txt

```diff
@@ -293,6 +293,18 @@ if(${BUILD_ROCAL})
     else()
         message(FATAL_ERROR "No filesystem library found.")
     endif()
+    
+    # Audio features for rocAL
+    if(${VX_EXT_RPP_VERSION_MAJOR} VERSION_GREATER_EQUAL "3" AND ${VX_EXT_RPP_VERSION_MINOR} VERSION_GREATER_EQUAL "0" AND ${VX_EXT_RPP_VERSION_PATCH} VERSION_GREATER "0")
+        set(AUDIO_SUPPORT TRUE)
```

Comment:
Also needs the deps to be installed

Reply:
It already checks for sndfile in line 308. Let me check why it doesn't check that if statement

Reply:
@kiritigowda pushed a change. Is CI down?
+ docker build -t compute-rocm-dkms-no-npi-hipclang-13897/ubuntu22gfx90a/math-ci:step1 --pull -f dockerfile-build-ubuntu22 --build-arg library_dependencies= --build-arg user_uid=1152 --build-arg base_image=10.216.51.87:5000/rocm-plus-docker/compute-rocm-dkms-no-npi-hipclang:13897-ubuntu-22.04-stg1 --build-arg build_num=13897 --cpuset-cpus=48,112,49,113,50,114,51,115,52,116,53,117,54,118,55,119,56,120,57,121,58,122,59,123,60,124,61,125,62,126,63,127 .

DEPRECATED: The legacy builder is deprecated and will be removed in a future release.

            Install the buildx component to build images with BuildKit:

            https://docs.docker.com/go/buildx/



Sending build context to Docker daemon  80.26MB


Step 1/15 : ARG base_image

Step 2/15 : FROM ${base_image}

error parsing HTTP 400 response body: invalid character '<' looking for beginning of value: "<html>\r\n<head><title>400 The plain HTTP request was sent to HTTPS port</title></head>\r\n<body>\r\n<center><h1>400 Bad Request</h1></center>\r\n<center>The plain HTTP request was sent to HTTPS port</center>\r\n<hr><center>nginx/1.23.4</center>\r\n</body>\r\n</html>\r\n"

script returned exit code 1